argus/config_generator/windows/cb_init.py

```diff
@@ -36,9 +36,17 @@ class BasePopulatedCBInitConfig(base.BaseWindowsConfig):
     def __init__(self, client):
         super(BasePopulatedCBInitConfig, self).__init__(client)
 
-    def set_conf_value(self, name, value=""):
-        """Set a config value in default section."""
-        self.conf.set("DEFAULT", name, value)
+    def set_conf_value(self, name, value="", section="DEFAULT"):
+        """Set a config value in the specified section."""
+        if section == "DEFAULT":
+            self.conf.set(section, name, value)
+            return
+        elif self.conf.has_section(section):
+            self.conf.set(section, name, value)
```

Comment:
Not really, structured the logic this way because of [this](http://pythonhosted.org/theape/infrastructure/api/ConfigParser.SafeConfigParser.has_section.html#ConfigParser.SafeConfigParser.has_section)